packages/antd/src/LabelAnt.tsx

```diff
@@ -45,7 +45,7 @@ export class BindMarkdownDiv extends React.Component<{ text: string } & React.HT
         const { text, ...props } = this.props;
         const html = marked(text ? text : '')
             .replace(/^<p>/, '')
-            .replace(/<\/p>$/, '');
+            .replace(/<\/p>/, '');
```

Comment:
the `/<\/p>$/` has removed the last `</p>` the new code removes the first `</p>` which may not be what you want. We could use `/<\/p>\s+$/` to deal with extra whitespace a the end. The we should  also use `/^\s+<p>/` for the beginning to deal with whitespaces at the beginning